### PR TITLE
Add section on episode 0

### PR DESCRIPTION
**Pull request by svenvanderburg**
Adds episode 0 to the course website. I wasn't able to build the website locally (see #11), so I'm not 100% sure that this renders nicely.

Relates to https://github.com/neural-data-science-course/research-software-development/pull/1

### CODE REVIEW

File: material.Rmd [hunk 1]
```diff
@@ -17,7 +46,7 @@ https://github.com/neural-data-science-course/neural-data
 
 
 
-#### [Local field potential (LFP)](https://github.com/neural-data-science-course/neural-data/blob/master/03-calcium_imaging/calcium_imaging.ipynb)
+#### [Local field potential (LFP)](https://github.com/neural-data-science-course/neural-data/blob/master/01-local_field_potential/local-field-potential.ipynb)
```
Comment (svenvanderburg) [inline material.Rmd:49]: This seems to be the correct link.

File: material.Rmd [hunk 2]
```diff
@@ -7,6 +7,35 @@ output:
 ---
 ---
 
+## Optional module 0: Good practices in Research Software Development.
```
Comment (davidespalla) [inline material.Rmd]:
```suggestion
## Module 0 (optional): Good practices in Research Software Development.
```

File: material.Rmd [hunk 3]
```diff
@@ -7,6 +7,35 @@ output:
...
+
```
Comment (davidespalla) [inline material.Rmd]:
```suggestion
This module is based on material from [The Carpentries](https://carpentries.org/), where you can find a lot of free, open source and high quality material on software development and data science.
```

**Review by davidespalla — COMMENTED** [review]
Hi Sven, changes looks great, I made a couple of suggestions to have a slightly more uniform module title and to explicitly acknowledge the carpentries. Let me know what you think.

**Comment by svenvanderburg** [comment]
Nice suggestions! I applied them :)

**Pull request merged**